### DISCUSSION TITLE
New Ticket Issue Category

### DISCUSSION THREAD

**Discussion by Tsearo** [RetroAchievements/RAWeb#779] [Ideas]
I think it could be helpful to add a category for tickets related to the Title and Descriptions. Something for things like typos, wording or inaccurate descriptions.

Idea came after reading the forum topic: Helping with language barriers
http://retroachievements.org/viewtopic.php?t=9927&c=55732

![image](https://user-images.githubusercontent.com/4047771/71909488-25322c80-313e-11ea-8997-7cc816079d35.png)

**Comment by Jamiras** [comment]
Related: https://github.com/RetroAchievements/RAIntegration/discussions/760#discussioncomment-1214934
